core/config/chaintype.go

```diff
@@ -1,4 +1,4 @@
-package chains
+package config
```

Comment:
This just avoids some import complexity from config having to import chains. There are only constant strings in here anyways.